src/styles/style.css

```
@@ -25,7 +25,7 @@ layer(base);
   content: none;
 }
 
-@plugin "daisyui" {
+@plugin "daisyui/index.js" {
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@plugin" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)